packages/cloudflare/package.json

```diff
@@ -59,7 +59,7 @@
     "vitest": "catalog:"
   },
   "dependencies": {
-    "@opennextjs/aws": "https://pkg.pr.new/@opennextjs/aws@5c0e121",
+    "@opennextjs/aws": "https://pkg.pr.new/@opennextjs/aws@642",
```

Comment:
@vicb are you sure that this is the correct package?

it seems to 404 😕 

https://pkg.pr.new/@opennextjs/aws@642:
![Screenshot 2024-11-22 at 10 42 23](https://github.com/user-attachments/assets/3103672e-6bd8-4d7f-a62f-6a5221db883b)

Reply:
(this is causing the prerelease of this PR to also not work)

Reply:
@Aslemammad any idea? 🙂

Reply:
It has been merged, maybe they delete it once merged. You could use https://pkg.pr.new/@opennextjs/aws@main instead

Reply:
Oops, sorry. 
I'll update the PR later today.

Reply:
Should be updated to https://pkg.pr.new/@opennextjs/aws@0ac604e

Reply:
forced push the updated.